applications/osb-portal/src/pages/RepositoryPage.tsx

```diff
@@ -186,14 +189,25 @@ const useStyles = makeStyles((theme) => ({
         flexGrow: 1,
       },
       "& .preview-box": {
+        backgroundColor: "rgba(255, 255, 255)",
```

Comment:
Black is not what we need here. A transparent background will be ok

Reply:
I had made the background white with the rgba(255, 255, 255). But when I change it to transparent, this is how it looks. I'm a little confused. Do you want black text on a white background or the other way around?

![pr1_repository_description](https://user-images.githubusercontent.com/40673427/117489695-eb448680-af3b-11eb-90e6-dc9503a5525e.PNG)